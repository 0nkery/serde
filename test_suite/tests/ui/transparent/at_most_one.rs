#[macro_use]
extern crate serde_derive;

#[derive(Serialize)]
#[serde(transparent)]
struct S {
    a: u8,
    b: u8,
}

fn main() {}
