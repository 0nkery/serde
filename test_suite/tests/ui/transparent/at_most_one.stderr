error: #[serde(transparent)] requires struct to have at most one transparent field
 --> $DIR/at_most_one.rs:5:1
  |
5 | / #[serde(transparent)]
6 | | struct S {
7 | |     a: u8,
8 | |     b: u8,
9 | | }
  | |_^

error: aborting due to previous error

